opslevel/resource_opslevel_property_assignment.go

```diff
@@ -139,11 +139,12 @@ func (resource *PropertyAssignmentResource) Read(ctx context.Context, req resour
 	owner := planModel.Owner.ValueString()
 	assignment, err := resource.client.GetProperty(owner, definition)
 	if err != nil {
+		if (assignment == nil || assignment.Definition.Id == "") && opslevel.IsOpsLevelApiError(err) {
```

Comment:
propery assignments don't have ID's - they are an assign / upsert kind of API - we don't need this.  In fact we should probably fully remove this error checking.